frontend/package.json

```diff
@@ -19,6 +19,7 @@
     "eslint-config-airbnb-typescript": "^17.0.0",
     "jest": "^29.0.0",
     "keycloak-js": "^19.0.3",
+    "moment": "^2.29.4",
```

Comment:
Why add moment? Some date helper or anything like that?

Reply:
Yes, we have some validations to do on datepicker, so we use the moment to help on that. 
Those validations are in 'frontend/src/components/SeedlotRegistrationProgress/InterimStorage/index.tsx'